### PR TITLE
fix lars optitmizer bug

### PR DESCRIPTION
**Pull request by firestonelib**
### PR types

Bug fixes
### PR changes

OPs
### Describe

Fix [lars optimizer](https://arxiv.org/pdf/1708.03888.pdf) lars_weight_decay=0 bug: 
When lars_weight_decay=0, we got local_learning_rate = learning_rate * lars_coeff * ||param|| / ||gradient||, but the formula in lars_momentum_op.cu when lars_weight_decay =0 equals to local_learning_rate = learning_rate.

image

### CODE REVIEW

**Review by JamesLim-sy — DISMISSED** [review]
LGTM

**Review by wangxicoding — DISMISSED** [review]
LGTM

**Review by jzhang533 — DISMISSED** [review]
LGTM

**Review by XiaoguangHu01 — DISMISSED** [review]
LGTM

**Review by TCChenlong — DISMISSED** [review]

**Review by fuyinno4 — APPROVED** [review]
